src/control/cmd/dmg/auto_test.go

```diff
@@ -0,0 +1,92 @@
+//
```

Comment:
Would it make more sense to call this and auto.go "config_test.go" and "config.go," respectively? Just wondering if there are more config-related pieces we might want to put into dmg, and might fit in well in these files.

Reply:
I had named as that previously but after discussion with @mjmac changed to auto.go